### PR TITLE
feat: cognito working example

### PR DESCRIPTION
**Pull request by rinzler99**

### CODE REVIEW

File: examples/cognito-react-example/config-overrides copy.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const webpack = require("webpack");
```
Comment (himanshuchawla009) [inline examples/cognito-react-example/config-overrides copy.js:1]: remove this file

File: examples/cognito-react-example/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- Go to identity providers tab under federation tab and select google. It will require you to enter your google's app client id and secret which you can obtain [here](https://console.cloud.google.com/apis/dashboard).
+NOTE: For creating google's app client id you need to select any one service from availabel services [Guide](https://aws.amazon.com/premiumsupport/knowledge-center/cognito-google-social-identity-provider/)
+
+- While configuring your google app oauth client for web, make sure to enter your `cognito hosted ui domain` in the `Authorized Javascript Origins list` and `<AWS_COGNITO_HOSTED_UI_DOMAIN>/oauth2/idpresponse` end point in the `Authorized redirect URIs list`.
+
+- Go back to your aws console and enter your google app client id and secret key.
+
+- Now your google login provider is configured, you can enable it in your App client settings in aws console.
+
+- Now go to your hosted app domain , it should show you hosted ui with google login button.
```
Comment (himanshuchawla009) [inline examples/cognito-react-example/README.md:38]: implementation is different from readme, we are not using cognito hosted UI right, we are using custom ui.  Please update the steps in readme accordingly.

File: examples/cognito-react-example/src/services/web3auth.tsx [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+        const adapter = new OpenloginAdapter({ adapterSettings: { 
+          network: web3AuthNetwork, 
+          clientId, 
+          uxMode: "redirect",
+          loginConfig: {
+            jwt: {
+              name: "Custom cognito Login",
+              verifier: "cognito-demo",
+              typeOfLogin: "jwt",
+              clientId: "29343gi24opt30vsa5s7u8gfvg",
```
Comment (himanshuchawla009) [inline examples/cognito-react-example/src/services/web3auth.tsx:120]: add code comment here about client id.

File: examples/cognito-react-example/src/services/web3auth.tsx [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+        uiConsole("web3auth not initialized yet");
+        return;
+      }
+      const localProvider = await web3Auth.connectTo(adapter, { 
+        relogin: true,
+        loginProvider,
+        extraLoginOptions: {
+          domain: "https://web3auth.auth.us-west-2.amazoncognito.com/oauth2",
+          verifierIdField: "email",
+          response_type: "token",
```
Comment (himanshuchawla009) [inline examples/cognito-react-example/src/services/web3auth.tsx:150]: add code comment about each variable here.

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

**Comment by himanshuchawla009** [comment]
there is some flicker while login, don't show logged in view on clicking of login button as shown in video:-

https://user-images.githubusercontent.com/14052374/168779037-2ac9b071-d807-4ae3-9f3b-fa2107589789.mov

File: examples/cognito-react-example/README.md (hunk 2 above)
Comment (rinzler99) [reply examples/cognito-react-example/README.md:38]: we are using hosted UI for aws cognito not for web3auth, i think this is right.

Comment (himanshuchawla009) [reply examples/cognito-react-example/README.md:38]: can you add the screenshot of hosted ui.

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**